bodhi/client/__init__.py

```diff
@@ -132,6 +132,7 @@ def _warn_if_url_and_staging_set(ctx, param, value):
     click.option('--state', type=click.Choice(['disabled', 'pending', 'current',
                                                'archived']),
                  help='The state of the release'),
+    click.option('--mail-template', help='Name of the email template for this release'),
     staging_option,
     url_option]
 
```

Comment:
The CLI should also display the mail template to the user when a release is printed to the terminal.